python/pyspark/sql/session.py

```diff
@@ -221,6 +221,9 @@ def __init__(self, sparkContext, jsparkSession=None):
                 or SparkSession._instantiatedSession._sc._jsc is None:
             SparkSession._instantiatedSession = self
 
+    def _repr_html_(self):
+        return self.sparkContext._repr_html_()
```

Comment:
Ok, I've set the `catalogImplementation` here now. This seems like something we could put in the SparkContext HTML repr as well.